src/components/inputs/rich-text-input/plugins/mark.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { isKeyHotkey } from 'is-hotkey';
+import invariant from 'tiny-invariant';
+import memoize from 'lodash/memoize';
+
+const memoizedIsHotkey = memoize(isKeyHotkey);
+
+const requiredOptions = [
+  'typeName',
+  'hotkey',
+  'command',
+  'query',
+  'RenderMark',
+];
+
+const validate = options => {
+  // eslint-disable-next-line consistent-return
+  const missingRequiredOptions = requiredOptions.filter(
+    option => !options[option]
+  );
+  return missingRequiredOptions;
+};
+
+const MarkPlugin = (options = {}) => {
+  const missingRequiredOptions = validate(options);
+
+  if (missingRequiredOptions.length > 0) {
```

Comment:
Hm. Why inside the `if`? Isn't this `invariant(missingRequiredOptions <= 0, ....)`